addons/l10n_it_edi/models/res_partner.py

```diff
@@ -21,6 +21,65 @@ class ResPartner(models.Model):
               associated with the office which, within the addressee administration, deals\
               with receiving (and processing) the invoice.")
 
+    l10n_it_edi_in_eu = fields.Boolean(string="Partner is in the EU group", compute="_compute_l10n_it_edi_in_eu", store=False)
```

Comment:
@lordkrandel No need to use fields for this as we do not need it in the views.  A simple method might do the trick.

Reply:
Sure! 👍🏻